packages/engine/lib/execution/src/package/init/js_py/message.rs

```diff
@@ -5,8 +5,11 @@ use stateful::agent::Agent;
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub enum JsPyInitTaskMessage {
+    #[serde(rename = "StartMessage")]
```

Comment:
I think we might just want to change the code on the other end (i.e. in the package language impls) because this isn't going to be ergonomic to do on all package message types